LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2017-2020 Denis Fäcke
+Copyright (c) 2017-2024 Denis Fäcke
```

Comment:
This is incorrect. We should add a new line with "The Fastify Contributors" instead, starting from the date of fork.